buildScript/gwt.gincl

```diff
@@ -167,7 +167,7 @@ task devMode (dependsOn: loadConfig) {
   outputs.upToDateWhen { false }
 
   doLast {
-    def res = project.ext.NODE 'yarn', 'run', 'dev', "--", "--baseWarName=${webapp.baseWarName}", "--app-name=${project.ext["app-name"]}"
+    def res = project.ext.NODE 'yarn', 'run', 'dev', "--", "--env.baseWarName=${webapp.baseWarName}", "--env.app-name=${project.ext["app-name"]}"
```

Comment:
I agree with @tgoldina that this whole file should be cleaned up.  She is planning to do that.